app/models/article.rb

```diff
@@ -22,6 +22,7 @@ class Article < ApplicationRecord
   counter_culture :organization
 
   has_many :comments, as: :commentable, inverse_of: :commentable
+  has_many :profile_pins, as: :pinnable, inverse_of: :pinnable
```

Comment:
I'd add a `dependent: :destroy` here, if the article gets removed, the pins should too